source/errors/404.html.erb

```diff
@@ -1 +1,7 @@
-<h1>404 Not Found</h1>
+<main id="content" class="app-c-content" role="main">
+  <div class="govuk-o-site-width-container app-c-site-width-container--constraint">
+    <h1 class="heading-xlarge">Page not found</h1>
+    <p>If you entered a web address please check it was correct.</p>
+    <p>You can find all of the current <a href="/design-patterns/patterns/styles">Styles</a>, <a href="/design-patterns/patterns/components">Components</a> and <a href="/design-patterns/patterns/patterns">Patterns</a> from browsing the navigation above.</p>
```

Comment:
I'd suggest either omitting this paragraph or at least the links for now - the links we end up with are unlikely to have the `/design-patterns/patterns` prefix and I worry we'll forget to come back and fix them…